episodes/01-introduction.md

```diff
@@ -1,101 +1,94 @@
 ---
-title: "Model Coupling"
-teaching: 10
-exercises: 2
+title: "Introduction to model coupling"
+teaching: 30
+exercises: 20
 ---
 
 :::::::::::::::::::::::::::::::::::::: questions 
 
-- How do you write a lesson using Markdown and `{sandpaper}`?
+- What is meant by a "model"?
+- What is the difference between a mathematical model and a submodel?
```

Comment:
Is this the difference you want to answer? After reading the rest of the episode I know the difference between mathematical model and computer model, but the relation to the concept of submodel is not so clear. That seems to be linked more closely to the concept of "coupled model", am I right?

Reply:
Ah good catch, sorry. No, this was a mistake in the question - I'll correct that now.